.eslintrc.js

```diff
@@ -244,7 +244,7 @@ module.exports = {
       },
     },
     {
-      files: ['packages/*/src/**/*{.ts,.tsx,.js}'],
+      files: ['packages/*/src/**/*.?(c|m)[jt]s?(x)'],
       excludedFiles: ['*.d.ts', '*.spec.ts', '*.spec.tsx'],
```

Comment:
I propose as a pattern we use `?(c|m)[jt]s?(x)` to select files, but use wildcard to exclude.

```suggestion
      excludedFiles: ['*.d.ts', '*.spec.*'],
```